packages/contracts/contracts/marketplace/V01_Marketplace.sol

```diff
@@ -310,14 +311,18 @@ contract V01_Marketplace is Ownable {
             );
         }
         require(offer.status == 2, "status != accepted");
-        paySeller(listingID, offerID); // Pay seller
-        if (msg.sender == offer.buyer) { // Only pay commission if buyer is finalizing
-            payCommission(listingID, offerID);
-        } else {
+        delete offers[listingID][offerID];
+
+        if (msg.sender != offer.buyer) {
```

Comment:
Any reason we swapped these around?

Reply:
`listing.deposit += offer.commission;` is an effect (state change). As per CEI, this effect should happen before interactions with other contracts (which `payCommission` and `paySeller` do)